ngraph/frontend/frontend_manager/include/frontend_manager/frontend_manager.hpp

```diff
@@ -93,6 +93,9 @@ struct FrontEndPluginInfo {
 };
 
 }  // namespace frontend
+}  // namespace ngraph
```

Comment:
BTW, I think we have move all frontend related stuff to OV namespace w/o compatibility headers.

They were not released

Reply:
I would like to do it in the separate PR